docs/source/getting_started.rst

```diff
@@ -0,0 +1,170 @@
+
+Welcome to the Arduino ESP32 documentation! Here you will find important information on how to use the project.
+
+First Things First
+------------------
+
+.. note::
+    Before continuing, we must be clear that this project is supported by `Espressif Systems`_ and the community.
+    Everyone is more than welcome to contribute back to this project.
+
+ESP32 is a single 2.4 GHz Wi-Fi-and-Bluetooth SoC (System On a Chip) designed by `Espressif Systems`_.
+
+ESP32 is designed for mobile, wearable electronics, and Internet-of-Things (IoT) applications. It features all the state-of-the-art characteristics 
+of low-power chips, including fine-grained clock gating, multiple power modes,and dynamic power scaling. For instance, in a low-power IoT sensor 
+hub application scenario, ESP32 is woken-up periodically and only when a specified condition is detected. Low-duty cycle is used to minimize the 
+amount of energy that the chip expends. 
+
+The output of the power amplifier is also adjustable, thus contributing to an optimal trade-off between communication range, data rate and 
+power consumption.
+
+The ESP32 series is available as a chip or module.
+
+Supported SoC's
+---------------
+
+Here are the ESP32 series supported by the Arduino-ESP32 project:
+
+======== ====== =========== ===================================
+SoC      Stable Development Datasheet
+======== ====== =========== ===================================
+ESP32    Yes    Yes         `ESP32 Datasheet`_
+ESP32-S2 No     Yes         `ESP32-S2 Datasheet`_
+ESP32-C3 No     Yes         `ESP32-C3 Datasheet`_
+ESP32-S3 No     No          Not Available Yet
+======== ====== =========== ===================================
+
+Supported Operating Systems
+---------------------------
+
++-------------------+-------------------+-------------------+
+| |windows-logo|    | |linux-logo|      | |macos-logo|      |
++-------------------+-------------------+-------------------+
+| Windows           | Linux             | macOS             |
++-------------------+-------------------+-------------------+
+
+.. |windows-logo| image:: _static/logo_windows.png
+.. |linux-logo| image:: _static/logo_linux.png
+.. |macos-logo| image:: _static/logo_macos.png
+
+Support
+*******
+
+This is a community project and it's supported by the community. Fell free to ask for help in one of the community channels.
+
+Issues Reporting
+****************
+
+Before opening a new issue, please read this first: 
+
+Be sure to search for a similar reported issue. This is to avoid duplicating or creating noise in the GitHub Issues reporting.
+We also have the troubleshooting guide to save your time on the most common issues reported by users.
+
+Community
+*********
+
+The Arduino community is huge! You can find a lot of useful content on the Internet.
+Here are some community channels where you may found information and ask for some help, if needed.
+
+- `ESP32 Forum`_: Official Espressif Forum for ESP32 related discussions.
```

Comment:
What about adding link directly to Arduino thread on ESP32 Forum?
[https://www.esp32.com/viewforum.php?f=19]

Reply:
Done!